wikimedia/wikipedia-ios
# Update text from 'nearly' to 'over' '300 languages'.

**Fixes Phabricator ticket:** 
https://phabricator.wikimedia.org/T243082

### Notes
Updated three spots the text 'nearly 300 languages' appears.
Updated code as well as strings files.

### Test Steps
- In onboarding, new text should appear:
    - On first screen in body text
    - On Search screen in title text
- To see in Search popup:
    - Add a second language in Settings or during onboarding
    - Switch to Search tab
    - The updated string will appear in the popover that explains the More button

### Screenshots/Videos *(if applicable)*
